x/examples/outline-connectivity/main.go

```diff
@@ -98,8 +132,26 @@ func main() {
 	domainFlag := flag.String("domain", "example.com.", "Domain name to resolve in the test")
 	resolverFlag := flag.String("resolver", "8.8.8.8,2001:4860:4860::8888", "Comma-separated list of addresses of DNS resolver to use for the test")
 	protoFlag := flag.String("proto", "tcp,udp", "Comma-separated list of the protocols to test. Must be \"tcp\", \"udp\", or a combination of them")
+	reportToFlag := flag.String("report-to", "", "URL to send JSON error reports to")
+	portFlag := flag.String("port", "53", "Resolver port to use for the test")
```

Comment:
Let's remove this flag for now, or add the port to the resolverFlag list instead.

Reply:
addressed in 69f4e9278e8c54cb69e750fca0e7b3a7d5859fbb